
takeoff_x: 0.0
takeoff_y: 0.0
takeoff_z: 0.5

center_of_circle_x: 1.0
center_of_circle_y: 1.0
center_of_circle_z: 0.5

touch_point_x: 0.0
touch_point_y: 1.0
touch_point_z: 0.5

